src/extra/AutoUI/Filters/FocusSearch.tsx

```diff
@@ -0,0 +1,146 @@
+import React from "react";
+import { Txt } from "../../../components/Txt";
+import { Checkbox } from "../../../components/Checkbox";
+import reject from "lodash/reject";
+import groupBy from "lodash/groupBy";
+import styled from "styled-components";
+import { Box } from "../../../components/Box";
+import { Flex } from "../../../components/Flex";
+import { useHistory } from "../../../hooks/useHistory";
+import { AutoUIContext } from "../schemaOps";
+import { stopEvent } from "../../../utils";
+import { AutoUIModel } from "..";
+
+const Focus = styled(Box)`
+	flex-basis: 500px;
+	background-color: white;
+	border: solid 1px ${(props) => props.theme.colors.quartenary.dark};
+	max-height: 200px;
+	position: absolute;
+	width: 100%;
+	z-index: 1;
+	border-radius: 0 0 ${(props) => props.theme.global.drop.border.radius}
+		${(props) => props.theme.global.drop.border.radius};
+	overflow: hidden;
+`;
+
+const FocusContent = styled(Box)`
+	max-height: 180px;
+	overflow-y: auto;
+	overflow-x: auto;
+`;
+
+const FocusItem = styled(Box)<{ hasGetBaseUrl: boolean }>`
+	cursor: ${(props) => (props.hasGetBaseUrl ? "pointer" : "default")};
+	&:hover {
+		background: #dde1f0; // This is the background color Select uses for entities on hover. We do not have it in our theme
```

Comment:
I'm ok if we do it in another PR but let's add it in the theme or we will soon probably make a mistake

Reply:
This is from the Focus Search PR, not a change from this PR. Agreed though, we should do so in a separate PR sooner than later. Only thing is I don't know if we want to keep using grommet's Select's hover effect color rather than using a different hover effect color of our choice